src/measures/fermions/ellesettete.cpp

```diff
@@ -108,11 +108,11 @@ namespace nissa
 				SUMM_THE_TRACE_PRINT_AT_LAST_HIT(Tr_three_pts,SIMPLE_PROP,SEQ_PROP);
 				
 				//////// disconnected //////// 
-				apply_stag_op(g5_source,conf,theory_pars->backfield[iflav],source);
+				apply_stag_op(g5_source,conf,theory_pars->backfield[iflav],15,15,source);
 				
 				//MINV(G5_PROP,iflav,g5_source); 
 				SUMM_THE_TIME_TRACE_PRINT_AT_LAST_HIT(Tr_first_bubble, SEQ_PROP, g5_source);
```

Comment:
Tr_first_bubble is a complex vector whereas the macro calls a routine which expects a double vector, you must change one of the two